### PR TITLE
[BUGFIX] BART and mBART support 2D attention mask from tokenizer

### PR DESCRIPTION
**Pull request by gongel**
### PR types

Bug fixes
### PR changes

Others

### CODE REVIEW

File: paddlenlp/transformers/mbart/modeling.py [hunk 1]
```diff
@@ -286,7 +286,13 @@ def forward(self, input_ids=None, attention_mask=None, **kwargs):
             attention_mask = paddle.cast(
                 input_ids == self.pad_token_id,
                 dtype=paddle.get_default_dtype()).unsqueeze([1, 2]) * -1e4
-            attention_mask.stop_gradient = True
+
+    # For 2D attention_mask from tokenizer
```
Comment (ZeyuChen) [inline paddlenlp/transformers/mbart/modeling.py]: 注释的位置不对

Comment (gongel) [reply paddlenlp/transformers/mbart/modeling.py]: Done and thx.

**Review by guoshengCS — APPROVED** [review]
